src/languageService/kustoLanguageService.ts

```diff
@@ -450,18 +450,11 @@ class KustoLanguageService implements LanguageService {
         return Promise.as([ls.TextEdit.replace(fullDocRange, formattedDoc)]);
     }
 
+    // Method is not triggered, instead doRangeFormat is invoked with the range of the caret's line.
     doCurrentCommandFormat(document: ls.TextDocument, caretPosition: ls.Position): Promise<ls.TextEdit[]> {
-        const positionResponse = { v: 0 };
-        const script = this.parseDocumentV2(document);
-        if (script.TryGetTextPosition(caretPosition.line, caretPosition.character, positionResponse)) {
-            const command = script.GetBlockAtPosition(positionResponse.v);
-            const start = document.positionAt(command.Start);
-            const end = document.positionAt(command.End);
-            const range = ls.Range.create(start, end);
-            return this.doRangeFormat(document, range);
-        }
-
-        return Promise.as(null);
+        const offset = document.offsetAt(caretPosition);
+        const range = this.createRange(document, offset - 1, offset + 1);
```

Comment:
why the extra offset? what happens if offset is 0? what happens if it's after last character?

Reply:
Without this change, if the cursor is at the end of a command, it won't prettify it. 

In this image, if the cursor is at the end of the line it won't work without this change:
![image](https://user-images.githubusercontent.com/8294298/88091941-92d70080-cb44-11ea-934f-4e9cdbbfc3f6.png)

Same if the cursor is in the beginning of the line. 

Regardless, this doesn't really matter since this method isn't really called.